cmd/tls-certs-check/main.go

```
@@ -52,7 +53,6 @@ func init() {
 
 func main() {
 	if err := rootCmd.Execute(); err != nil {
-		log.WithError(err).Fatal("error in the cli. Exiting")
```

Are we sure that removing this we don't fail silently when we shouldn't? if I miss-type something in the command for example

---

What does silent error is the `SilenceUsage` configuration parameter of the `rootCmd` struct.
This `log.Fatal` sentence is just a redundant error message.

---

excellent! thanks

---

@angelbarrera92 it is not quite clear to me how this is a redundant message. for example: https://github.com/sighupio/tls-certs-check/blob/095b60ffdb92fe6825b4537fbe01ba0f232414e1/pkg/tls-certs-check/main.go#L63 seems to return an error which is simply returned by `TLSCheck` function. shouldn't we be logging in out somewhere?

---

The `checkCertificateDate` function returns an error only if the certificate is going to expire in a time window defined from the `days` function argument.
The `TLSCheck` just checks if the certificate is experired.